README.md

```diff
@@ -77,6 +77,14 @@ func main() {
 	// by a call to the Eval method.
 	prob := anom.Push(8.0)
 	fmt.Println("Anomalous Probability:", prob)
+
+  // PushFixed method will keep the size of the Data vector constant.
+  // Oldest data points will be evicted as points are added.
+  // WARNING: Mixing Push() and PushFixed() will result in failure!
```

Comment:
If `Push()` and `PushFixed()` no longer conflict, maybe remove this warning and the one below, right before the definition of `PushFixed()`.